doc/rst/technotes/initequals.rst

```diff
@@ -272,3 +272,38 @@ type:
 
 If the value is not a subtype of the generic expression, then there will be a
 compile-time error. This may change in future releases.
+
+
+Disabling Copyability
+---------------------
+If a user wishes to indicate that their record cannot be copied, they can do
+so by implementing an ``init=`` method with a ``false`` where-clause:
+
+.. code-block:: chapel
+
+  proc R.init=(other: R) where false {
+   // method body may be empty in this case
+  }
+
+The ``compilerError`` utility function can be used for the same purpose:
+
+.. code-block:: chapel
+
+  proc R.init=(other: R) {
+    compilerError("Cannot copy R");
+  }
+
+Relation to Assignment Operator
+-------------------------------
+
+In the 1.20 release users could choose to implement either the ``init=`` method
+or ``=`` operator for a given type, or implement both, or rely entirely on the
+compiler-generated implementation. This could lead to hard-to-debug problems
+when both functions appeared to be user-defined, but a user mistake in the
+function signature caused the compiler-generated version to be used instead.
```

Comment:
Maybe add "caused *it to be ignored* and the compiler-generated version ..." to be as explicit as possible?